README.md

```diff
@@ -54,6 +54,13 @@ Thanks goes to these wonderful people ([:hugs:](https://allcontributors.org/docs
                     <sub><b>Ali Solanki</b></sub>
                 </a> 
             </td>
+            <td align="center">
+                <a href="https://github.com/karann09">
+                    <img src="https://avatars.githubusercontent.com/u/126327364?v=4" width="100px;" alt="Karann09"/>
```

Comment:
I suggest using Karan Kumar instead of Karann09 because it is a more accessible name.
```suggestion
                    <img src="https://avatars.githubusercontent.com/u/126327364?v=4" width="100px;" alt="Karan Kumar"/>
```